src/groovy/org/transmartproject/db/dataquery/highdim/tworegion/TwoRegionModule.groovy

```diff
@@ -0,0 +1,126 @@
+package org.transmartproject.db.dataquery.highdim.tworegion
+
+import grails.orm.HibernateCriteriaBuilder
+import org.hibernate.ScrollableResults
+import org.hibernate.engine.SessionImplementor
+import org.hibernate.transform.Transformers
+import org.springframework.beans.factory.annotation.Autowired
+import org.transmartproject.core.dataquery.TabularResult
+import org.transmartproject.core.dataquery.highdim.AssayColumn
+import org.transmartproject.core.dataquery.highdim.projections.Projection
+import org.transmartproject.db.dataquery.MultiTabularResult
+import org.transmartproject.db.dataquery.highdim.AbstractHighDimensionDataTypeModule
+import org.transmartproject.db.dataquery.highdim.chromoregion.TwoChromosomesSegmentConstraintFactory
+import org.transmartproject.db.dataquery.highdim.parameterproducers.AllDataProjectionFactory
+import org.transmartproject.db.dataquery.highdim.parameterproducers.DataRetrievalParameterFactory
+
+/**
+ * Created by j.hudecek on 4-12-2014.
+ */
+class TwoRegionModule extends AbstractHighDimensionDataTypeModule {
+
+    final String name = 'two_region'
+
+    final String description = "Two  Variant data"
+    final String TWO_REGION_PROJECTION = "two_region_label"
+
+    final List<String> platformMarkerTypes = ['two_region']
+
+    final Map<String, Class> dataProperties = typesMap(DeTwoRegionJunction,
+            ['upChromosome', 'downChromosome', 'id', 'upEnd', 'upPos', 'upStrand', 'downEnd', 'downPos', 'downStrand', 'isInFrame'])
+
+    final Map<String, Class> rowProperties = typesMap(JunctionsRow,
+            ['allJunctions'])
```

Comment:
Will have to be changed accordingly, possibly to an empty list. Difficult to tell without knowing what a row actually represents.